[hi] Localized pod.md

oops-shlok opened the pull request:
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

oops-shlok committed:
[hi] Localized pod.md

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

aj11anuj reviewed (changes requested):
Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

oops-shlok committed:
[hi] Fixed Core hindi words

Signed-off-by: Shlok Upadhyay <deep.upu9@gmail.com>

oops-shlok commented:
> Same mistake, [refer this comment](https://github.com/cncf/glossary/pull/3443#pullrequestreview-2760283037)

I have fixed it now and updated the code. Please review once.

aj11anuj commented on content/hi/pod.md:
```diff
@@ -0,0 +1,31 @@
+
+पॉड्स ऐसे कंटेनर्स को एक साथ ग्रुप कर देते हैं जो आपस में जुड़े हुए हैं, जिससे ऑपरेशन आसान हो जाता है।  
+उदाहरण के लिए, कोई कंटेनर जो मुख्य कंटेनर को सेटिंग्स देता है,  या कोई _साइडकार_ कंटेनर जो नेटवर्क ट्रैफिक हैंडल करता है (देखें [सर्विस मेश](/service-mesh/)),
+या फिर कोई कंटेनर जो लॉग्स कलेक्ट करता है, ये सभी पॉड के अंदर एक साथ रखे जा सकते हैं।  
```
Fluency of reading is changing in this paragraph, fix it. Also make sure to change the links to redirect to hindi localized pages not english ones.

bishal7679 commented:
@oops-shlok Gentle Reminder for this one as well!

mailnike reviewed (commented):
Thanks for localizing pod.md. The translation is clear overall. One sentence in the **समस्या** section reads a little awkwardly: `उदाहरण के लिए, ऑपरेटर को यह तय करने के लिए कि वे एक साथ रहें, ये कंटेनर्स एक साथ ही डिप्लॉय हों।` Consider rephrasing for clarity, for example: `उदाहरण के लिए, यह सुनिश्चित करने के लिए कि वे एक साथ रहें, ऑपरेटर को इन कंटेनर्स को एक साथ डिप्लॉय करना पड़ता है।`
